docs/source/datasets.rst

```diff
@@ -1,8 +1,10 @@
 torchtext.datasets
-====================
+==================
 
 .. currentmodule:: torchtext.datasets
 
+TorchText datasets.
```

Comment:
How is this going to be rendered? Is this meant to be a title?

Reply:
It's just a text. Not title. I will remove it.